src/webapp/components/issues/IssueFilters.tsx

```diff
@@ -181,11 +188,16 @@ export type IssueFilterState = {
     periods: Period[];
     status: Maybe<Id[]>;
     step: Maybe<Id[]>;
+    search: Maybe<string>;
 };
 
 const FilterContainer = styled.div`
-    padding: 0 1rem;
     align-items: center;
     display: flex;
     gap: 1rem;
+    padding: 0 5px;
```

Comment:
shortcuts apply physical properties, which can create issues with different writing modes. We should use the padding shortcut when all four sides have the same value. Otherwise, it's recommended to use separate padding-inline and padding-block properties to ensure proper formatting and consistency. In this case:
```
padding-block: 0;
padding-inline: 5px;
```